contributing.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+`Functional.CQS.AOP` is intended to be software-agnostic.  As such, library enhancements and modifications must be discussed first.  Please [log an issue](https://github.com/RyanMarcotte/Functional.CQS/issues) prior to writing any code.
```

Comment:
Perhaps this is a phrase that exists in other sources, but I'm not sure if "software-agnostic" clearly conveys the intended meaning? Would "consumer-agnostic" or "client-agnostic" work?

If I'm reading this correctly, I assume you're trying to say that enhancements should be general-purpose, not specific to the needs of a particular client/consumer of the libraries.

To me, "software-agnostic" could be unclear, because I think, "Well, isn't this library also software?"

Reply:
"application-agnostic"?

Reply:
Ah, that works for me!

Reply:
Done